xarray/core/dataset.py

```diff
@@ -3150,11 +3150,25 @@ def drop(self, labels, dim=None, *, errors='raise'):
             any of the variable or index labels passed are not
             in the dataset. If 'ignore', any given labels that are in the
             dataset are dropped and no error is raised.
+        **dim_kwargs : {dim: label, ...}, optional
+            The keyword arguments form of ``dim`` and ``labels``.
+            Either ``dim`` and ``labels`` or ``dim_kwargs`` must be provided.
 
         Returns
         -------
         dropped : Dataset
```

Comment:
I would be awesome if you're up for adding a few examples here inline in the docstring.

Reply:
Good idea. Done.